prac_05/hex_colours.py

```diff
@@ -0,0 +1,14 @@
+"""Look up hexadecimal colour codes"""
```

Comment:
The question doesn't specifically ask for it (though it says "based on the previous question"), but adding some formatting to make it more readable would be a nice addition.